File: dynamic/Maruko.Dynamic.Config/Internal/Service/DTO/PageConfigDTO.cs
using System;
using System.Collections.Generic;
using System.Text;
using Maruko.Core.Application.Servers.Dto;
using Maruko.Core.AutoMapper.AutoMapper;

namespace Maruko.Dynamic.Config
{
    [AutoMap(typeof(PageConfig))]
    public class PageConfigDTO : EntityDto
    {
        public long PageId { get; set; }
        public string Key { get; set; }
        public string DataUrl { get; set; }
        public bool IsRow { get; set; }
        public bool IsMultiple { get; set; }
        public int RowWith { get; set; }
        public string Fields { get; set; } = "";
        public string Buttons { get; set; } = "";
        public string Functions { get; set; } = "";
        public bool IsTableOperateRow { get; set; }
        public bool IsTableCheckRow { get; set; }
    }
}

File: act/act-modules/act-modules-zero/src/main/java/com/act/modules/zero/internal/application/pageConfig/dto/PageConfigDTO.java
package com.act.modules.zero.internal.application.pageConfig.dto;

import com.act.core.domain.BaseEntity;
import lombok.Data;

@Data
@SuppressWarnings("all")
public class PageConfigDTO extends BaseEntity<Long> {
    private long pageId;
    private String key;
    private String dataUrl;
    private Boolean isRow = false;
    private Boolean isMultiple = false;
    private int rowWith = 0;
    private String fields;
    private String buttons;
    private String functions = "";
    private Boolean isTableOperateRow = false;
    private Boolean isTableCheckRow = false;
}
